framework_crates/bones_schema/macros/src/lib.rs

```diff
@@ -366,38 +373,43 @@ pub fn derive_has_schema(input: TokenStream) -> TokenStream {
     };
 
     if let Some(generic_params) = input.generic_params() {
-        let mut sync_send_generic_params = generic_params.clone();
-        for (param, _) in sync_send_generic_params.params.iter_mut() {
-            let clone_bound = if !no_clone { quote!(+ Clone) } else { quote!() };
```

Comment:
lmk if I'm missing anything - but I think we may have dropped the `#[schema(no_clone]` (i.e. no_clone bool) support here, if this attribute is present, we should not be requiring clone. If reading this right - a no_clone type with generic params will now require clone on its generic types, which previously did not - might not compile with the right instantiation. (we may not have any test coverage or usage that would break in code I'm guessing).

Reply:
Really good catch. Fixed, and added a test that will fail to compile if the code gen is broken again.